pkg/asset/templates/kubelet.yaml

```diff
@@ -17,27 +17,19 @@ spec:
       - name: kubelet
         image: quay.io/coreos/hyperkube:v1.3.0_coreos.1
         command:
-        - /nsenter
-        - --target=1
-        - --mount
-        - --wd=.
-        - --
-        - ./hyperkube
-        - kubelet
-        - --api-servers={{ index .APIServers 0 }}
-        - --config=/etc/kubernetes/manifests
-        - --allow-privileged
-        - --hostname-override=$(MY_POD_IP)
-        - --cluster-dns=10.3.0.10
-        - --cluster-domain=cluster.local
-        - --kubeconfig=/etc/kubernetes/kubeconfig
-        - --lock-file=/var/run/lock/kubelet.lock
-        - --minimum-container-ttl-duration=3m0s
+        - /bin/bash
+        - -c
+        - "/nsenter --target=1 --mount --wd=. -- ./hyperkube kubelet --hostname-override=$(MY_POD_IP) $(FLAGS)"
         env:
           - name: MY_POD_IP
             valueFrom:
               fieldRef:
                 fieldPath: status.podIP
+          - name: FLAGS
+            valueFrom:
+              configMapKeyRef:
+                name: kubelet-config
```

Comment:
Well.. we already didn't have that option - so maybe this isn't a big deal.

So the upstream route which we can eventually use is likely going to allow "global config" vs "node-named config" .. which we can piggy back on.

In the interim, what do you think about using the on-disk checkpointed config? That way, whatever config is defined by the node-annotation is what we use (even in the self-hosted copy).